routes/projectRoutes.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express');
+const router = express.Router();
+const projectDb = require('../data/helpers/projectModel');
+
+router.get('/', async (req, res) => {
+  try {
+    const projects = await projectDb.get();
+    res.status(200).json(projects);
+  } catch (error) {
+    res.status(400).json({
+      errorMessage: 'Unable to get projects.'
+    });
+  }
+});
+
+router.get('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const projects = await projectDb.get(id);
+    projects === undefined
+      ? res.status(404).json({
+          message: 'The project with the specified ID does not exist.'
+        })
+      : res.status(200).json(projects);
+  } catch (error) {
+    res.status(400).json({
+      errorMessage: 'Unable to get users.'
+    });
+  }
+});
+
+router.get('/getactions/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const projects = await projectDb.getProjectActions(id);
+    res.status(200).json(projects);
+  } catch (error) {
+    res.status(400).json({
+      errorMessage: 'Unable to get projects.'
+    });
+  }
+});
+
+router.post('/', async (req, res) => {
+  try {
```

Comment:
I really love that async await syntax. The majority are using promises but this is so much cleaner and the more modern way to write promises. (although async/await is using that promise syntax under the hood)